client/cli/ok.py

```diff
@@ -27,82 +71,85 @@
-
-    # Autostyle
-    parser.add_argument('--style', action='store_true',
-                        help="Run AutoStyle Feedback System")
-    # Submission Export
-    parser.add_argument('--export', action='store_true',
-                        help="Downloads all submissions for the current assignment")
-    parser.add_argument('--latest', action='store_true',
-                        help="When used with --export, downloads latest submissions instead of final submissions")
+    testing.add_argument('--submit', action='store_true',
+                        help="submit the assignment")
+    testing.add_argument('--backup', action='store_true',
+                        help="attempt to reliably backup your work")
+    testing.add_argument('--revise', action='store_true',
+                        help="submit composition revision")
+    testing.add_argument('--restore', action='store_true',
+                        help="restore assignment from an earlier backup")
+    testing.add_argument('--timeout', type=int, default=10,
+                        help="set the timeout duration (in seconds) for running tests")
+
+    # Experiments
+    experiment = parser.add_argument_group('experiment options')
+    experiment.add_argument('--no-hints', action='store_true',
+                        help="do not give hints")
+    experiment.add_argument('--hint', action='store_true',
+                        help="give a hint (if available)")
+    experiment.add_argument('--style', action='store_true',
+                        help="run AutoStyle feedback system")
 
     # Debug information
-    parser.add_argument('--version', action='store_true',
-                        help="Prints the version number and quits")
-    parser.add_argument('--tests', action='store_true',
+    debug = parser.add_argument_group('debugging options')
+    debug.add_argument('--version', action='store_true',
+                        help="print the version number and exit")
+    debug.add_argument('--tests', action='store_true',
                         help="display a list of all available tests")
-    parser.add_argument('--debug', action='store_true',
-                        help="show debug statements")
+    debug.add_argument('--debug', action='store_true',
+                        help="show debugging output")
+
+    # Grading
+    grading = parser.add_argument_group('grading options')
+    grading.add_argument('--lock', action='store_true',
+                        help="lock the tests in a directory")
+    grading.add_argument('--score', action='store_true',
+                        help="score the assignment")
+    grading.add_argument('--score-out', type=argparse.FileType('w'),
+                        default=sys.stdout, help="write scores to a file")
+    grading.add_argument('--config', type=str,
+                        help="use a specific configuration file")
+    grading.add_argument('--export', action='store_true',
```

Comment:
`--export` is broken for v3. Can we remove it for now? Hopefully, we'll get it back at some point.